docs/UserGuide.md

```diff
@@ -306,6 +306,22 @@ Example:
 
 `note 2 r/loves green` will create a note under the contact at index 2 that reads `loves green`
 
+### Viewing a contact's full details : `view`
+
+Allows you to view the full details of the contact, as some are hidden in the contact list.
+
+Format `view INDEX`
+
+#### Commands in detailed view
+
+When viewing a contact in detailed view, some commands become available, and commands that work in list view may either still work, or will not work.
+
+In general, commands for modifying a contact will work, and will modify the contact currently displayed. As such, there is no need to give an index for those commands anymore, and they will be ignored if the command is called in this view.
+
+If the command does not work in list view, the app will inform you. To return to list view, use `list` or other commands that produces a list of contacts.
+
```

Comment:
It's good that you're documenting this down in the UG for our users! However (minor nit) we could probably rephrase this, or rethink the way we want to deliver this to the users. 

To elaborate, we could probably rephrase some of the words used like:
> some commands become available, and comamnds that work...

... might be better to rephrase it something along the lines of "some commands in detailed view are different from those in list view", then explicitly laying out a table of detailed-related commands. Personally the current phrasing is (inherently, by nature of the feature) confusing and I feel like it will be better if we explain it through examples/glossary rather than through elaboration.

Furthermore, and I think this is more important, we could probably be more explicit in the view-mode that users are put into. For example, we have a small text label somewhere on the window to indicate which view-mode they are in. This becomes a key-feature in our app, then it becomes a lot easier to explain to users what commands they can make in which modes. Let me know what you think!

Reply:
Yea I had a little bit of a tough time thinking of the simplest way to put it. I'll include your suggestion as well as what I proposed.